qcfractal/interface/collections/torsiondrive_dataset.py

```diff
@@ -12,7 +12,7 @@
 from .collection_utils import register_collection
 
 if TYPE_CHECKING:  # pragma: no cover
-    from ..models import Molecule  # lgtm[py/unused-import] (https://github.com/Semmle/ql/issues/2014)
```

Comment:
It looks like autoflake is unaware of `if TYPE_CHECKING` in the same way that LGTM was. I guess we're going to have to substitute the lgtm annotation for a pyflakes annotation...

Reply:
Maybe turn off `auto flake` for now? Its nice, but not needed.